test/adapters/http_server.spec.js

```diff
@@ -276,7 +276,7 @@ describe('read http_server', function() {
             return finish;
         })
         .then(function(results) {
-            expect(results.errors).to.deep.equal(['Invalid CSV data: Error: Row length does not match headers']);
+            expect(results.errors).to.deep.equal(['Invalid CSV data: Red Dragon, 24.04 at line 2']);
```

Comment:
Not as easy to get the headers as I had thought so if the error happens on the second line I don't have the headers (since no valid point was processed till that point) and would simply write out 

```
Invalid CSV data: "Red Dragon, 24.04" at line 2 does not match header line
```

So I'm thinking I'll just print that instead (without the actual header line)